### PR TITLE
Trying to fix merge from tekton-jx-orchestration branch

### PR DESCRIPTION
**Pull request by dcanadillas**
Changed some files and their references from the post

### CODE REVIEW

File: content/posts/tekton-jx-pipelines.md [hunk 1]
```diff
@@ -0,0 +1,954 @@
+---
+author:
+  name: "David Cañadillas"
+title: "Jenkins X Orchestration: More than Tekton on Steroids"
+date: 2019-07-20T19:39:10+01:00
```
Comment (kmadel) [inline content/posts/tekton-jx-pipelines.md]: I would just update the date to today so your post is at the top of the list. Otherwise looks good!

Comment (dcanadillas) [reply content/posts/tekton-jx-pipelines.md]: So true!

**Comment by kmadel** [comment]
Could we get one more review before publishing - please :)

**Comment by kmadel** [comment]
@dcanadillas is this ready from your point of view - and if so, we need one more review, should be quick.

**Comment by dcanadillas** [comment]
> @dcanadillas is this ready from your point of view - and if so, we need one more review, should be quick.

/lgtm

File: content/posts/tekton-jx-pipelines.md [hunk 2]
```diff
@@ -0,0 +1,954 @@
...
+
+Decoupling and scaling pipelines is one of the best approaches for building new modern applications and microservices. But it can be very hard to work on these decoupled objects to define a pipeline, which conceptually is a sequential and "monolithic" tasks execution.
+
+So, Tekton is demonstrating its power of decoupling CI/CD pipelines and builds to scale. But it also needs a powerful orchestrator to simplify the complexity underneath, and even more when dealing with a platform like Kubernetes, that can also add more complexity.
+
+I like to say that the best way to build and run CI/CD pipelines for modern (and also traditional, why not!) software applications is to make it simple and "abstract the abstraction". I believe Jenkins X is about that. It is about applying the simplicity of trational Jenkins pipelines from a powerful and scalable engine.
```
Comment (logandonley) [inline content/posts/tekton-jx-pipelines.md]: > It is about applying the simplicity of trational Jenkins pipelines

Typo - should be traditional

File: content/posts/tekton-jx-pipelines.md [hunk 3]
```diff
@@ -0,0 +1,954 @@
...
+
+Most development companies are realizing that CI/CD next iteration is already here and that it's not only about cloud native pipelines, YAML object definintions or Kubernetes containers orchestration. It is about orchestrating all the complexity while adopting best practices. We are talking about build packs templating, simple YAML pipelines, seamless Git integration, scalable team management, flexible and standard deployments, extensible platform engine, standard packaging and promotion, etc.
```
Comment (logandonley) [inline content/posts/tekton-jx-pipelines.md]: > realizing that CI/CD next iteration is already here

I think this should be reworded to:

realizing that the next iteration of CI/CD is already here

File: content/posts/tekton-jx-pipelines.md [hunk 4]
```diff
@@ -0,0 +1,954 @@
...
+
+We may know [Jenkins X](https://jenkins-x.io) as a new pure CI/CD cloud native implementation different than [Jenkins](https://jenkins.io). It is based on the use of Kubernetes Custom Resource Definitions (CRD's) to experience a seamless execution of CI/CD pipelines. This happens by leveraging the power of Kubernetes in terms of scalability, infrastructure abstraction and velocity.
+
+The new main approach of Jenkins X is about a serverless experience, because there is no more traditional Jenkins engine running. So, it relies on a CI/CD pipeline engine that can run on any standard Kubernetes deployment. This engine is the [Tekton CD project ](https://github.com/tektoncd/pipeline), a former Google project that is also - like Jenkins X - part of the [Continuous Delivery Foundation](https://cd.foundation/).
+
+In this post, we are showing the power of Tekton as a decoupled CI/CD pipeline engine execution. But more important, why an orchestration pipeline platform is practically required to design, configure and run your pipelines for the whole Software Delivery process. This orchestration platform is Jenkins X.
+
+## The Tekton base
+
+Why then Tekton is a cool CI/CD engine?
```
Comment (logandonley) [inline content/posts/tekton-jx-pipelines.md]: > Why then Tekton is a cool CI/CD engine?

I would rearrange this to:

Why then is Tekton a cool CI/CD engine?

File: content/posts/tekton-jx-pipelines.md [hunk 5]
```diff
@@ -0,0 +1,954 @@
...
+      name: petclinic-git
+  - name: docker-container
+    resourceRef:
+      name: petclinic-image
+  - name: deploy-repo
+    resourceRef:
+      name: petclinic-deploy
+```
+
+In the `PipelineRun` object is important to understand that is intended in the way of defining the specific running instance of a deployed pipeline. It's the right place to assign running input/output parameter values, specify `serviceAccounts` to be used, or referencing the right `PipelieResources` to be used.
```
Comment (logandonley) [inline content/posts/tekton-jx-pipelines.md]: > In the `PipelineRun` object is important to understand that is intended in the way

Missing a couple of 'it's in there, should be:

In the `PipelineRun` object it is important to understand that it is intended in the way

**Pull request merged**